ansible-scylla-node/tasks/RedHat.yml

```diff
@@ -62,7 +77,7 @@
         pre_reboot_delay: 0
         post_reboot_delay: 30
         test_command: whoami
-      when: reboot_after_kernel_upgrade|bool and kernel_installed.changed
+      when: (reboot_after_kernel_upgrade|bool and kernel_installed.changed) r restart_required.rc == 1
```

Comment:
OR ? (not r ? ) typo @Rumbles ?

Reply:
Hi, you're right there is a mistake here, but I ran in a to a larger issue... when I tested this method, the yum package absent step will fail as one of the kernels it tries to remove is the running kernel...

Reply:
ok, I think that should be better now